x/delayedack/ibc_middleware.go

```diff
@@ -46,7 +46,12 @@ func (im IBCMiddleware) OnRecvPacket(
 	if !im.keeper.IsRollappsEnabled(ctx) {
 		return im.IBCModule.OnRecvPacket(ctx, packet, relayer)
 	}
-	logger := ctx.Logger().With("module", "DelayedAckMiddleware")
+	logger := ctx.Logger().With(
```

Comment:
it has some duplicates with the 
`logger.Debug("Saving rollapp packet"`
that follows